.github/workflows/pkgci_shark_ai.yml

```diff
@@ -0,0 +1,95 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: PkgCI - shark-ai
+
+on:
+  workflow_call:
+    inputs:
+      artifact_run_id:
+        type: string
+        default: ""
+  workflow_dispatch:
+    inputs:
+      artifact_run_id:
+        type: string
+        description: "Id for a workflow run that produced dev packages"
+        default: ""
+
+jobs:
+  test_shortfin_llm_server:
+    name: "Integration Tests - Shortfin LLM Server"
+    strategy:
+      matrix:
+        version: [3.11]
+      fail-fast: false
+    runs-on: mi300x-4
```

Comment:
So much faster on an mi300 machine.

https://github.com/nod-ai/shark-ai/actions/runs/12701070231/job/35405010380?pr=780

Probably fine to stay there given the benefits, but I don't want _all_ of our tests running on the same machines. That makes it too easy to depend on hardcoded paths and persistent files.

Reply:
yeah; would be nice if we can eventually move as much as possible onto standard runners.